azurerm/internal/services/cosmos/tests/resource_arm_cosmosdb_sql_container_test.go

```diff
@@ -127,7 +129,7 @@ func testCheckAzureRMCosmosDbSqlContainerExists(resourceName string) resource.Te
 		resourceGroup := rs.Primary.Attributes["resource_group_name"]
 		database := rs.Primary.Attributes["database_name"]
 
-		resp, err := client.GetSQLContainer(ctx, resourceGroup, database, account, name)
+		resp, err := client.GetSQLContainer(ctx, resourceGroup, account, database, name)
```

Comment:
turns out the reason the tests were failing was because the inputs were incorrect. it passed before b/c until recently `database` and `account` used the same naming format.